RecoDataProducts/inc/HelixRecoDir.hh

```diff
@@ -18,6 +18,28 @@ namespace mu2e {
     float slopeSig() const { return std::fabs(_slope/_slopeErr); }
     float chi2ndof() const { return _chi2ndof; }
 
+    // Enum declaration for direction
+    enum PropDir {
+      upstream,
+      downstream,
+      ambiguous
+    };
+
+    // Method to compute direction based on slope and slopeSig
+    PropDir computeDirection(float sigThreshold) const {
```

Comment:
This function name is misleading, the direction (slope) isn't being computed, just compared. slopeSignificance?